build.gradle

```diff
@@ -178,6 +179,56 @@ subprojects { subproject ->
 										}
 							}
 
+							if (subproject.plugins.hasPlugin('org.jetbrains.kotlin.jvm')) {
+								def kotlinVersion = '2.2.21'
```

Comment:
Feels like this has to be to Gradle `ext` variable. 
See how we do that in Spring Integration: https://github.com/spring-projects/spring-integration/blob/main/build.gradle#L2